### PR TITLE
feat(install): Probe KubeArmor after installing 

### LINKED ISSUE #334: karmor install to check if the target env supports enforcement
Currently, the user explicitly has to run the `karmor probe` to check if the target env has enforcement capabilities. I got into a discussion today where the user couldn't figure why the `"Action: Audit (Block)"` was shown in the `karmor logs` output.

The user himself proposed a solution where if we can show after `karmor install` whether the target environment supports enforcement mode.

The enviroment used was CIVO with k3s cluster. I am not aware of the target distro used underneath k3s but I assume both AppArmor and BPF-LSM were not supported there.

CC: @IdealUsrname, @daemon1024

### PR DESCRIPTION
**Pull request by daemon1024**
Fixes #334 

Refactored Probe Code to return probe information before printing it
Leveraging the API then in Install to extract enforcement data 

Screenshot when Enforcement Works
image

When Enforcement doesn't work
image

### CODE REVIEW

File: k8s/env.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package k8s
```
Comment (nyrahul) [inline k8s/env.go:1]: license header

File: probe/probe.go [hunk 2]
```diff
@@ -90,10 +89,17 @@ func PrintProbeResult(c *k8s.Client, o Options) error {
 	if isKubeArmorRunning(c, o) {
 		getKubeArmorDeployments(c, o)
 		getKubeArmorContainers(c, o)
-		err := probeRunningKubeArmorNodes(c, o)
+		probeData, err := ProbeRunningKubeArmorNodes(c, o)
 		if err != nil {
 			log.Println("error occured when probing kubearmor nodes", err)
```
Comment (nyrahul) [inline probe/probe.go:94]: return here on err?

**Review by nyrahul — APPROVED** [review]
LGTM

**Review by DelusionalOptimist — COMMENTED** [review]
Looking mostly good. Just some nits.

**Review by DelusionalOptimist — APPROVED** [review]

**Pull request merged**